packages/office-ui-fabric-react/src/components/DatePicker/DatePicker.tsx

```diff
@@ -194,10 +194,18 @@ export class DatePicker extends BaseComponent<IDatePickerProps, IDatePickerState
       className,
       minDate,
       maxDate,
-      calendarProps
+      calendarProps,
+      calloutProps
```

Comment:
This is exposing/leaking the fact that the DatePicker uses a callout to consumers. Which props do you need to be able to set in the callout? It seems better to just expose the specific props in the component itself.

Reply:
I don't remember.  I will remove it.